### PR TITLE
readme update - refine usage part -2

### PR DESCRIPTION
**Pull request by peggy-ntt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +53,7 @@ Sets the plugin type
 
 ## Input Plugins
 
-You can configure the document input using `bcgov.access.input` property.
+You can configure the document input using `bcgov.access.input.plugin` property.
```
Comment (alexjoybc) [inline README.md]: right, you can add `the document  input plugin`

File: jrcc-access-spring-boot-autoconfigure/src/main/java/ca/bc/gov/open/jrccaccess/autoconfigure/plugins/console/DefaultPrettifier.java [hunk 2]
```diff
@@ -19,7 +19,9 @@
 	@Override
 	public String prettify(String input) {
 		int max = 100;
-		if(input.length() < max) max = input.length();
+		if(input.length() < max){
+			return input;
+		}
 		return input.substring(0, max) + " ...";
```
Comment (alexjoybc) [inline jrcc-access-spring-boot-autoconfigure/src/main/java/ca/bc/gov/open/jrccaccess/autoconfigure/plugins/console/DefaultPrettifier.java]: can you use `MessageFormater.format()`

Comment (peggy-ntt) [reply jrcc-access-spring-boot-autoconfigure/src/main/java/ca/bc/gov/open/jrccaccess/autoconfigure/plugins/console/DefaultPrettifier.java]: ok, I will change that.

File: jrcc-access-spring-boot-autoconfigure/src/main/java/ca/bc/gov/open/jrccaccess/autoconfigure/plugins/console/DefaultPrettifier.java [hunk 3]
```diff
@@ -22,7 +24,7 @@ public String prettify(String input) {
 		if(input.length() < max){
 			return input;
 		}
-		return input.substring(0, max) + " ...";
+		return MessageFormat.format("{0} ...", input.substring(0, max) );
```
Comment (alexjoybc) [inline jrcc-access-spring-boot-autoconfigure/src/main/java/ca/bc/gov/open/jrccaccess/autoconfigure/plugins/console/DefaultPrettifier.java]: Nice

**Pull request merged**